chromadb/test/property/test_add.py

```diff
@@ -179,22 +185,29 @@ def test_add_large(
         embedding_function=collection.embedding_function,
     )
     normalized_record_set = invariants.wrap_all(record_set)
-    initial_version = coll.get_model()["version"]
+    initial_version = cast(int, coll.get_model()["version"])
 
+    ids = []
```

Comment:
in `create_large_recordset`, `ids` is never `None` - if it were, `create_batches` would fail because it would call `len(None)`. Let's just be defensive and raise if `ids` turns out to be `None` on `RecordSet` in this case.